Rakefile

```diff
@@ -7,7 +7,7 @@
 #  Modifications Copyright OpenSearch Contributors. See
 #  GitHub history for details.
 
-require "logstash/devutils/rake"
+#require "logstash/devutils/rake"
```

Comment:
We can get rid of this commented out import.

Reply:
done